components/org.wso2.micro.integrator.security/src/main/java/org/wso2/micro/integrator/security/extensions/SecurityTokenStore.java

```diff
@@ -0,0 +1,6 @@
+package org.wso2.micro.integrator.security.extensions;
+
+import org.apache.rahas.SimpleTokenStore;
+
+public class SecurityTokenStore extends SimpleTokenStore{
```

Comment:
Incorrect formatting. Needs to have a space between SimpleTokenStore and {

Reply:
fixed